spec/selenium_spec.rb

```diff
@@ -100,6 +100,22 @@ module TestSessions
         expect(@session).to have_selector(:css, '.change_event_triggered', :match => :one)
       end
     end
+
+    describe "#path" do
+      before :each do
+        @session.visit('/with_js')
+      end
+
+      it "returns xpath" do
+        element = @session.find(:css, '#with_focus_event')
+        expect(element.path).to eq('/html/body/p[5]/input')
```

Comment:
Not every driver may produce exactly the same xpath - so its probably better to get the #path result, find the element using that xpath and then make sure they're the same element